tests/integration/targets/rds_cluster_modify/tasks/main.yaml

```diff
@@ -5,6 +5,12 @@
       secret_key: '{{ aws_secret_key }}'
       session_token: '{{ security_token | default(omit) }}'
   block:
+
```

Comment:
Don't we need group/amazon.cloud.aws under module_defaults for the amazon.cloud module to work?

Reply:
Had discussed this with Jill, it worked without `group/amazon.cloud.aws` after adding `amazon.cloud` to test requirements
https://github.com/ansible-collections/amazon.aws/pull/1705/files#diff-12ceb68f7a1282d864ebd30d4e5732b76b32fe7d3bdf77969043694f5b6346dfL3-R5